### ISSUE TITLE
Unary `+` parameter triggers overeager breaking of method chain

### ISSUE THREAD

**Issue by pushkine** [prettier/prettier#13018]
**Prettier 2.7.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzCEAUB9AhgSgDoAjXAJ0yIBs44ZKBuAHSnSwFoCTzLCa7GLNpgDUXUhWq16+BiAA0ICAAcYAS2gBnZKHJkIAdwAK5BNpS4qB3AE9ti4mVxgA1nQDKuALZwAMmqg4ZFRLTTgHJ1cPZWcAgHNkGDIAV3CQOC9iOAATbJzfXCg45Nw4uAAxCDIvXBh1IuQQXGSYCAUQAAsYLyoAdQ61eE0YsDh3M0G1ADdBm0awTXsQALCyGCMnOJrg0LSAK00AD3d4mgBFZIh4HaowxRiyVcbSLKp25TIAmF61bJgO5AADgADPd9GFek5lI0PnBVlMgooAI6XeAbFTmJqadiBHI5dpkOAotSEjalbZIEK3NJhLxqRIpGmnOAXK5BSm7RQwXDEH5-AFIABMXKcaio8QAwhAvBT0poAKztZJhAAqPPMVLuICmqQAklA8rB3GBPqoAIIG9wwGw0G5hAC+9qAA)

```sh
--parser babel
```

**Input:**

```jsx
foo(_a).bar().leet();
foo(-a).bar().leet();
foo(+a).bar().leet();
```

**Output:**

```jsx
foo(_a).bar().leet();
foo(-a).bar().leet();
foo(+a)
  .bar()
  .leet();

```

**Expected behavior:**
```jsx
foo(_a).bar().leet();
foo(-a).bar().leet();
foo(+a).bar().leet();
```

**Comment by t-mangoe** [comment]
I have looked into this issue.
It seems that the Doc created by prettier is different for `+` parameter and other cases.
This Doc is created at the end of `printMemberChain` function in `src/language-js/print/member-chain.js`

https://github.com/prettier/prettier/blob/a043ac0d733c4d53f980aa73807a63fc914f23bd/src/language-js/print/member-chain.js#L387-L405

In the above process, `isSimpleCallArgument` function is used in the if statement.
But, `+` unary expression does not appear to be properly processed in this function.

https://github.com/prettier/prettier/blob/a043ac0d733c4d53f980aa73807a63fc914f23bd/src/language-js/utils/index.js#L886-L891

I added `node.operator === "+"` in the above if state as a trial, and the output seemed to be expected.
(So, if you like, maybe I can send a PR about this issue.)
